<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>200</green>
        <blue>11</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>147</red>
        <green>255</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>113</red>
        <green>227</green>
        <blue>195</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>100</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>53</red>
        <green>133</green>
        <blue>111</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>200</green>
        <blue>11</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>200</green>
        <blue>11</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>227</green>
        <blue>210</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>200</green>
        <blue>11</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>147</red>
        <green>255</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>113</red>
        <green>227</green>
        <blue>195</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>100</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>53</red>
        <green>133</green>
        <blue>111</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>200</green>
        <blue>11</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>200</green>
        <blue>11</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>227</green>
        <blue>210</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>100</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>200</green>
        <blue>11</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>147</red>
        <green>255</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>113</red>
        <green>227</green>
        <blue>195</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>100</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>53</red>
        <green>133</green>
        <blue>111</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>100</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>100</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>200</green>
        <blue>11</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>200</green>
        <blue>11</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>80</red>
        <green>200</green>
        <blue>166</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>DVR管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;WenQuanYi Zen Hei&quot;;
background-color: rgb(112, 200, 11);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>200</green>
         <blue>11</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>200</green>
         <blue>11</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>200</green>
         <blue>11</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>83</red>
         <green>238</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>200</green>
         <blue>11</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>200</green>
         <blue>11</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>200</green>
         <blue>11</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>83</red>
         <green>238</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>200</green>
         <blue>11</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>200</green>
         <blue>11</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>200</green>
         <blue>11</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Highlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="tbtn_querylog">
         <property name="font">
          <font>
           <family>WenQuanYi Zen Hei</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>BlankCursor</cursorShape>
         </property>
         <property name="text">
          <string>文件管理</string>
         </property>
         <property name="icon">
          <iconset resource="../src/btnImage.qrc">
           <normaloff>:/manangertools/image/foudergroup.png</normaloff>:/manangertools/image/foudergroup.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbtn_Settings">
         <property name="font">
          <font>
           <family>WenQuanYi Zen Hei</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>BlankCursor</cursorShape>
         </property>
         <property name="text">
          <string>系统设置</string>
         </property>
         <property name="icon">
          <iconset resource="../src/btnImage.qrc">
           <normaloff>:/images/image/linkurl.png</normaloff>:/images/image/linkurl.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbtn_moveSettings">
         <property name="font">
          <font>
           <family>WenQuanYi Zen Hei</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>BlankCursor</cursorShape>
         </property>
         <property name="text">
          <string>媒体播放</string>
         </property>
         <property name="icon">
          <iconset resource="../src/btnImage.qrc">
           <normaloff>:/images/image/imovie.png</normaloff>:/images/image/imovie.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="tbtn_netSettings">
         <property name="font">
          <font>
           <family>WenQuanYi Zen Hei</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>BlankCursor</cursorShape>
         </property>
         <property name="text">
          <string>网络设置</string>
         </property>
         <property name="icon">
          <iconset resource="../src/btnImage.qrc">
           <normaloff>:/images/image/thurdbird.png</normaloff>:/images/image/thurdbird.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbtn_imageSettings">
         <property name="font">
          <font>
           <family>WenQuanYi Zen Hei</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>BlankCursor</cursorShape>
         </property>
         <property name="text">
          <string>capture</string>
         </property>
         <property name="icon">
          <iconset resource="../src/btnImage.qrc">
           <normaloff>:/images/image/preview.png</normaloff>:/images/image/preview.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbtn_managetools">
         <property name="font">
          <font>
           <family>WenQuanYi Zen Hei</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>BlankCursor</cursorShape>
         </property>
         <property name="text">
          <string>系统工具</string>
         </property>
         <property name="icon">
          <iconset resource="../src/btnImage.qrc">
           <normaloff>:/manangertools/image/spaces.png</normaloff>:/manangertools/image/spaces.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_ledAlarm">
    <property name="geometry">
     <rect>
      <x>808</x>
      <y>20</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>媒体播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ledRunning">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ledError">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>120</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>LED ERROR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setRelay1">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>170</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>SET RELAY1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setRelay2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>220</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>SET RELAY2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_setRelay1">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>170</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 234, 234);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_setRelay2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>220</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 234, 234);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_readRelay1">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>270</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Read RELAY1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_readRelay2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>310</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Read RELAY2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_relay2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>310</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 228, 228);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_relay1">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>270</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 228, 228);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../src/btnImage.qrc"/>
 </resources>
 <connections/>
</ui>
